.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "tailwind",
+          "apply",
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ]
+  },
+  "csstree/validator": true,
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+}
+
```

Comment:
## 2
### Your Summary
![pr-summary](https://user-images.githubusercontent.com/93039359/184694244-0c0e8dc9-1d85-4eda-b1d2-8eba1038c3f5.png)


### Descriptive Summary Example :heavy_check_mark:
![good-summary](https://user-images.githubusercontent.com/93039359/184694824-ded06807-d049-4ad8-9971-cd9b9d4143c0.png)


- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: